compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/misc/AnnotationsTest.java

```diff
@@ -82,4 +82,9 @@ public void testAnnotAttachmentAtEOF() {
     public void testAnnotAtExprLevelRecovery() {
         testFile("annotations/annotations_source_11.bal", "annotations/annotations_assert_11.json");
     }
+
+    @Test
+    public void testAnnotTupleMembers() {
```

Comment:
Since the new change is not for the [annotation](https://ballerina.io/spec/lang/master/#annotation) grammar, shall we move this to `TupleTypeTest` class?